lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -54,21 +57,25 @@ def define
         option :no
 
         precalculate :exempt_benefits do
-          config.exempt_benefits
+          config.exempt_benefits(chosen_cap.to_sym)
```

Comment:
It seems odd that in some places we're calling `String#to_sym` on `chosen_cap` and in others we are not. If this mixture of approaches is actually necessary, then the commit note should explain why. If it's not necessary, then I'd prefer all the method calls to pass `chosen_cap` consistently i.e. always pass as a string OR always pass as a symbol.